src/AppEntry.tsx

```diff
@@ -30,6 +30,10 @@ const AppEntry: React.FunctionComponent<AppEntryProps> = ({ logger }) => {
     }
   }, [logger]);
 
+  useEffect(() => {
+    insights?.chrome?.appAction?.('view-list-page');
+  }, []);
+
```

Comment:
This is the higher point in the tree that [this](https://github.com/content-services/content-sources-frontend/pull/17/files#diff-26dd5a05981860b950dac92a2279676793db35f257c4634be527ec13f928782bL30-L32) was placed